pkg/querier/timeseries_series_set.go

```diff
@@ -0,0 +1,89 @@
+package querier
+
+import (
+	"sort"
+
+	"github.com/cortexproject/cortex/pkg/ingester/client"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/storage"
+)
+
+// timeSeriesSeriesSet is a wrapper around a client.TimeSeries slice to implement to SeriesSet interface
+type timeSeriesSeriesSet struct {
+	ts []client.TimeSeries
+	i  int
+}
+
+// Timeseries is a type wrapper that implements the storage.Series interface
```

Comment:
timeSeriesSeriesSet type method should be here, not later in the file.

Reply:
moved.